.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.9"
\ No newline at end of file
+  - "0.10.33"
+before_script:
+  - "npm install -g jshint mocha"
```

Comment:
These can go in `devDependencies` — npm will copy these modules' executables into `node_modules/.bin`, and will look there when running scripts.